# Input
# 0 - end log
# 1 - start log
bool IO

# Log file name 
string file_name

# Parameter to log
string[] parameter
---
bool success